builtins/davix/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ foreach(lib davix neon)
 endforeach()
 
 ExternalProject_Add(DAVIX
-  URL ${DAVIX_URL}/${DAVIX_VERSION}/davix-embedded-${DAVIX_VERSION}.tar.gz
+  URL ${DAVIX_URL}/davix-${DAVIX_VERSION}.tar.gz
```

Comment:
See c2ee81c98741c93ea71ee7677129b185332f6b11 - this might require passing `-DDAVIX_TESTS=OFF` to Davix's CMake configuration.

Reply:
I've added `-DDAVIX_TESTS=OFF` to Davix's CMake configuration.